packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -190,7 +207,7 @@ export const getPolymarketProposedPriceRequestsOO = async (
       })
       .filter((event) => {
         const requestId = ethers.utils.keccak256(
-          ethers.utils.defaultAbiCoder.encode(
+          ethers.utils.solidityPack(
```

Comment:
Not technically a bug, since we only compare against requestIds generated offchain.  But I figured it was probably good practice to follow the same request id generation method used in the contracts everywhere.